src/ReportIrradiation.cc

```diff
@@ -137,6 +137,18 @@ void ReportIrradiation::visualizeTo(RootWSite& site) {
   mapNamesInfo = new RootWInfo("Dose and irradiation map names");
   mapNamesInfo->setValue(mapNames_);
   settingsContent.addItem(mapNamesInfo);
+  RootWInfo* notesInfo;
+  notesInfo = new RootWInfo("Note:");
```

Comment:
Thanks for adding the notes (and cleaning up the bits of information I added above, it looks much nicer on the page now!) - if I may nitpick, the `xyzInfo` name and value are always separated by a colon when printed on the webpage, so I think can just write `new RootWInfo("Note")` here.